java/jdbc/src/main/java/io/vitess/jdbc/VitessMySQLDatabaseMetadata.java

```diff
@@ -67,16 +67,16 @@ public class VitessMySQLDatabaseMetadata extends VitessDatabaseMetaData implemen
         "INNER", "INOUT", "INSENSITIVE", "INSERT", "INT", "INT1", "INT2", "INT3", "INT4", "INT8",
         "INTEGER", "INTERVAL", "INTO", "IS", "ITERATE", "JOIN", "KEY", "KEYS", "KILL", "LEADING",
         "LEAVE", "LEFT", "LIKE", "LIMIT", "LINEAR", "LINES", "LOAD", "LOCALTIME", "LOCALTIMESTAMP",
-        "LOCK", "LONG", "LONGBLOB", "LONGTEXT", "LOOP", "LOW_PRIORITY", "MATCH", "MEDIUMBLOB",
+        "LOCK", "LONG", "LONGBLOB", "LONGTEXT", "LOOP", "LOW_PRIORITY", "MANUAL", "MATCH", "MEDIUMBLOB",
```

Comment:
the whole list needs re-formatting, for the CI to pass. line wrapping at 100 chars required.

Reply:
Fixed!